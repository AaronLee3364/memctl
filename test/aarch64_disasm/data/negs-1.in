fffffff00654df4c 6b1503e8
