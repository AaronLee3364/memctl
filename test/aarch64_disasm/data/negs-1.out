fffffff00654df4c  NEGS    W8, W21
